toolchain/parse/node_kind.def

```diff
@@ -226,10 +246,8 @@ CARBON_PARSE_NODE_KIND_CHILD_COUNT(ContinueStatement, 1,
 
 // `return`:
 //   ReturnStatementStart
-//   _external_: expression
+//   _optional_ _external_: expression
```

Comment:
The way this reads feels a little odd to me, maybe remove the underscores instead?

Reply:
SGTM, maybe something for me to think about but I started this with `_external_`